OrbitGl/CaptureWindow.cpp

```diff
@@ -489,16 +489,16 @@ void CaptureWindow::KeyPressed(unsigned int a_KeyCode, bool a_Ctrl,
         break;
       case 18:  // Left
         if (a_Shift) {
-          time_graph_.OnShiftLeft();
+          time_graph_.JumpToBox(TimeGraph::Previous, TimeGraph::SameFunction);
```

Comment:
I would consider keeping the names OnShiftLeft, OnLeft etc. and then have them be a simple implementation like 

void OnShiftLeft() {
  JumpToBox(TimerGraph::Next, TimeGraph::SameFunction);
}

etc. 

In particular, if you keep the OnUp() and OnDown() methods (as below).

Reply:
Ahh, I'm sorry. This was my idea. But I see your point with OnUp/OnDown.

Reply:
I think either way is fine, but would prefer consistency. 

Either keep the old names or rename OnUp/OnDown in a way consistent with the new names. 

I think that TimeGraph should not necessarily explicitly mention the buttons that had to be pressed to get a certain action  (OnLeft, OnRight), so I would prefer to rename OnUp/OnDown.

Reply:
> I think that TimeGraph should not necessarily explicitly mention the buttons that had to be pressed to get a certain action (OnLeft, OnRight), so I would prefer to rename OnUp/OnDown.

Yes, that was also my thought.

Reply:
Yes, was the original plan, but I preferred to start with the change of Left/Right to see opinions.

Reply:
What do you think now?